code/src/Pizza.ts

```diff
@@ -0,0 +1,24 @@
+import { Consumable } from './Consumable';
+
+export class Pizza extends Consumable {
+  numberOfSlices: number;
+  slicesEaten: number = 0;
+  constructor(numberOfSlices: number, spoiled: boolean) {
+    super('Pizza', 0, 0, spoiled);
+    this.numberOfSlices = numberOfSlices;
+  }
+
+  eat(): string {
+    if (this.slicesEaten < this.numberOfSlices) {
+      this.slicesEaten++;
+
+      if (this.slicesEaten >= this.numberOfSlices) {
+        this.consumed = true;
```

Comment:
It's a good practice to use setter for this, especially considering that you have it in your Consumable class (setConsumed).

Reply:
By the way, this code example in the homework file contains a mistake, can you find it?